packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-spinner-container .pt-spinner-head {
```

Comment:
disagree, i don't think we should support red spinners inside blue buttons, for instance. things are always white inside an intent.